### DISCUSSION TITLE
How about add an action to update a update.json file?

### DISCUSSION THREAD

**Discussion by 117649** [onemen/TabMixPlus#280] [Ideas]
update.json is something like this:
https://github.com/p1usminus/roomybookmarksredux/blob/master/update.json

I think it let can FF check the update by itself if its url is included in install.rdf .

**Comment by onemen** [comment]
currently I am running a script on my PC to do the release.
I just call it with the version number and it:
- update.json file
- add commit for release with tag
- generate the change log
- create the release on Github
- push xpi copy to [bitbucket](https://bitbucket.org/onemen/tabmixplus-for-firefox/downloads/)

The action I've added today will automatically release new dev-build on every commit to the main branch (it still have some bugs)
I also added action to update the new doc [change-log](https://onemen.github.io/tabmixplus-docs/releases/latest/) on every release
